config/routes.rb

```diff
@@ -0,0 +1,61 @@
+
+
+Rails.application.routes.draw do
+  root 'media_ranker#index'
+
+  resources :media_ranker
```

Comment:
If we're only using the `index` action from this controller, we could say:
```ruby
resources :media_ranker, only: [:index]
```
or actually just stick with the root route definition from line 4.